sklearn/metrics/_plot/cap_curve.py

```diff
@@ -0,0 +1,313 @@
+import numpy as np
+
+from ...utils._plotting import _BinaryClassifierCurveDisplayMixin
+
+
+class CumulativeAccuracyDisplay(_BinaryClassifierCurveDisplayMixin):
```

Comment:
```suggestion
class CapCurveDisplay(_BinaryClassifierCurveDisplayMixin):
```
This name would align with `RocCurveDisplay` and `DetDurveDisplay`.

Site remark: I would have preferred capital letters for acronyms, i.e. `ROCCurveDisplay` and `CAPCurceDisplay`, but this ship has sailed.

Reply:
I also would have preferred capital letters for acronyms... Maybe it's not too late to avoid propagating a bad convention to new names? We could also rename and alias the old names with a silent deprecation.